spec/requests/workflows/create_workflow_spec.rb

```diff
@@ -70,39 +86,53 @@
       allow(SendUpdateMessage).to receive(:publish)
     end
 
-    context 'when the version exists' do
+    context 'when the version is passed' do
       it 'creates new workflows' do
         expect do
-          post "/objects/#{druid}/workflows/#{workflow}"
+          post "/objects/#{druid}/workflows/#{workflow}?version=1"
         end.to change(WorkflowStep, :count).by(16)
         expect(WorkflowStep.last.lane_id).to eq('default')
         expect(SendUpdateMessage).to have_received(:publish).with(druid: druid)
       end
 
       it 'sets the lane id' do
-        post "/objects/#{druid}/workflows/#{workflow}?lane_id=#{lane_id}"
+        post "/objects/#{druid}/workflows/#{workflow}?lane_id=#{lane_id}&version=1"
         expect(WorkflowStep.last.lane_id).to eq(lane_id)
       end
 
       context 'and the request is bad' do
         let(:workflow) { 'xaccessionWF' }
         it 'returns a 400 error' do
           expect do
-            post "/objects/#{druid}/workflows/#{workflow}"
+            post "/objects/#{druid}/workflows/#{workflow}?version=1"
           end.not_to change(WorkflowStep, :count)
           expect(response.status).to eq 400
         end
       end
     end
 
-    context "when the version doesn't exist" do
+    context 'when the version exists in dor-services-app (deprecated)' do
```

Comment:
add version is not passed, pls.

Reply:
It wouldn't check with dor-services-app if the version were passed so that is implied.

Reply:
okay, maybe for all these tests of deprecated ways, the tests will be going away once we know we don't need to keep the deprecated methods around, right?  So maybe it's not so important that these descriptions are clear to the folks that don't have all the pieces clear in their heads.